ckanext/multilang/templates/package/edit_multilang.html

```diff
@@ -0,0 +1 @@
+{% extends "package/edit.html" %}
```

Comment:
Is this file really needed? It only extends, but no overrides are implemented

Reply:
Yep, it seems it's unused. Removing..